### PR TITLE
feat: implementation of DID authentication

### PR DESCRIPTION
**Pull request by H4NLee**
For DID authentication, an empty presentation with proof is used ([ref](https://w3c-ccg.github.io/vp-request-spec/#did-authentication)).
User can verify the DID auth by using the `VerifyPresentation`.

### CODE REVIEW

File: pkg/vc/command_test.go [hunk 1]
```diff
@@ -229,6 +257,7 @@ func TestFullScenarioWithBBS(t *testing.T) {
 type MockVDR struct {
 	pubKeyBz   []byte
 	pubKeyType string
+	did        string
```
Comment (youngjoon-lee) [inline pkg/vc/command_test.go:260]: i think this one is not used anywhere

File: pkg/vdr/did_auth_test.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	panaceaVDR := NewPanaceaVDR(suite.VDR)
+	f, err := vc.NewFramework(panaceaVDR)
+	suite.NoError(err)
+
+	proofOpts := &vc.ProofOptions{
+		Controller:         suite.holderDID,
+		VerificationMethod: fmt.Sprintf("%s#key1", suite.holderDID),
+		SignatureType:      ecdsaSigType,
+		Domain:             suite.domain,
+		Challenge:          suite.challenge,
```
Comment (youngjoon-lee) [inline pkg/vdr/did_auth_test.go:86]: Could you also check if these `domain` and `challenge` affects the signature? It must be, if I understand correctly. But, IIRC, aries-framework-go doesn't handle domain and challenge properly to create a signature, from @audtlr24's research.

**Review by youngjoon-lee — COMMENTED** [review]
all goo

File: pkg/vdr/did_auth_test.go (hunk 2 above)
Comment (H4NLee) [reply pkg/vdr/did_auth_test.go:87]: I confirmed that different signatures are generated with different challenge and domain. ([3e470fb](https://github.com/medibloc/vc-sdk/pull/20/commits/3e470fb0ff5d97cf6d581ed37394434fbffe6552))

File: pkg/vc/command_test.go (hunk 1 above)
Comment (H4NLee) [reply pkg/vc/command_test.go]: I missed it to delete. Thanks

File: pkg/vdr/did_auth_test.go (hunk 2 above)
Comment (youngjoon-lee) [reply pkg/vdr/did_auth_test.go:87]: great. danke.

**Review by youngjoon-lee — APPROVED** [review]
lgtm

**Review by audtlr24 — APPROVED** [review]
LGTM, thanks!

**Review by gyuguen — APPROVED** [review]
LGTM

**Review by inchori — APPROVED** [review]
LGTM! 👍

**Pull request merged**